docs/cli/user-manual.md

```diff
@@ -265,7 +265,7 @@ Manage the configuration data either in user scope or project scope.
 |:----------------  |:-------------|:-------------|
 |`--env`| Yes | Select an existing environment for the project. |
 |`--folder`| No | Project directory. This is used when get/set project configuration. The default value is: `./`. |
-|`--global`| No | Cope of configuration. If this is true, the scope is limited to user scope instead of project scope. The default value is: `false`. Currently supported global configurations including: `telemetry`, `validate-dotnet-sdk`, `validate-func-core-tools`, `validate-node`. |
+|`--global`| No | Cope of configuration. If this is true, the scope is limited to user scope instead of project scope. The default value is: `false`. Currently supported global configurations including: `telemetry`, `validate-dotnet-sdk`, `validate-func-core-tools`, `validate-node`, `validate-ngrok`, `trust-development-certification`. |
```

Comment:
`certificate` or `certification`?
@MuyangAmigo - we provide the similar behavior as [dotnet dev-certs](https://docs.microsoft.com/en-us/dotnet/core/additional-tools/self-signed-certificates-guide#with-dotnet-dev-certs), which uses `certificate`.
<img width="841" alt="image" src="https://user-images.githubusercontent.com/7642967/151499909-4cfa05fb-7faa-4a83-99f3-a5843ee2e23f.png">

Reply:
Got it. Renamed it in PR: https://github.com/OfficeDev/TeamsFx/pull/3816